src/main/java/com/ecommerce/galleon/dto/cart/CartItemDto.java

```diff
@@ -0,0 +1,44 @@
+package com.ecommerce.galleon.dto.cart;
+
+
+import com.ecommerce.galleon.model.Cart;
+import com.ecommerce.galleon.model.Product;
+
+public class CartItemDto {
+    private Integer id;
+    private Integer quantity;
+    private Product product;
+
+    public CartItemDto() {
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public Integer getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(Integer quantity) {
+        this.quantity = quantity;
+    }
+
+    public Product getProduct() {
+        return product;
+    }
+
+    public void setProduct(Product product) {
+        this.product = product;
+    }
+
+    public CartItemDto(Cart cart) {
+        this.id = cart.getId();
+        this.quantity = cart.getQuantity();
+        this.setProduct(cart.getProduct());
```

Comment:
The assignment of quantity and setProduct. could you implement them in the same way ?